CONTRIBUTING.md

```diff
@@ -102,8 +102,8 @@ Guidelines](https://opensource.google.com/conduct/).
 gcloud services enable container.googleapis.com
 
 gcloud container clusters create demo --zone=us-central1-a \
-    --machine-type=n1-standard-2 \
-    --num-nodes=4 \
+    --machine-type=n1-standard-1 \
+    --num-nodes=3 \
```

Comment:
If I understand correctly, the intention in this PR was to fix #542 If so, you need to rebase your branch - it currently includes all the other changes you've been doing for other PRs. All the changes to minimize the cost or to move/remove docs should not be part of this PR.
This (rebase) is needed so that we have a clean history of changes, can rollback or port (if needed) every change separately.

Reply:
Changed in new branch